src/main/java/com/dev/cinema/config/AppConfig.java

```diff
@@ -0,0 +1,56 @@
+package com.dev.cinema.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {
+        "com.dev.cinema.dao.impl",
+        "com.dev.cinema.service.impl",
+        "com.dev.cinema.security"
+})
+public class AppConfig {
+    private final Environment environment;
+
+    public AppConfig(Environment environment) {
```

Comment:
A good practice is adding annotation @Autowired above

Reply:
I thought, that it is unnecessery if we have only one constructor, Waiting for menthor's comment :)